### PR TITLE
add KCD Paris 2020

### PR DESCRIPTION
**Pull request by jcsirot**
Initial PR for KCD Paris 2020. Please DO NOT MERGE

Details about the venue, the CFP and the sponsors are coming soon.

### CODE REVIEW

File: content/events/2020-paris/_index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: Paris
+date: '2020-10-29'
+end: '2020-10-30'
+location:
+  country: France
+  city: Paris
+  url: https://en.wikipedia.org/wiki/Paris
+  venue: "????"
```
Comment (KTan1226) [inline content/events/2020-paris/_index.md]: Hi folks! Can you please specify the venue?

File: content/events/2020-paris/_index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+map: paris-map.png
+sched_slug: kubernetescommunitydayparis
+social:
+  twitter: 'https://twitter.com/kubedaysparis'
```
Comment (KTan1226) [inline content/events/2020-paris/_index.md]: Just an FYI- The Twitter account looks good but is missing a profile image.

File: content/events/2020-paris/_index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  orgs:
+- level: gold
+  orgs:
+speakers:
+# - name: Phippy
+#   image: phippy-color.png
+#   twitter: https://twitter.com/cloudnativefdn
+#   bio: Phippy is a simple PHP app, trying to find a home in a cloud native world.
+#   homepage: https://phippy.io
+organizers:
```
Comment (KTan1226) [inline content/events/2020-paris/_index.md]: It looks like some of the images aren't showing up and I know you are still adding some as this is WIP. Please let me know when complete and we can address if there is a bug.

Comment (KTan1226) [reply content/events/2020-paris/_index.md]: Please also update the email address on this page to your alias. Thanks!

File: content/events/2020-paris/_index.md (hunk 1 above)
Comment (jcsirot) [reply content/events/2020-paris/_index.md]: We are still in the process of confirming the venue and the official date of the event. I'll update the PR asap.

File: content/events/2020-paris/_index.md (hunk 2 above)
Comment (jcsirot) [reply content/events/2020-paris/_index.md]: We are currently working with a designer to make a logo and a profile picture for the event.

File: content/events/2020-paris/_index.md (hunk 3 above)
Comment (jcsirot) [reply content/events/2020-paris/_index.md]: Sure, I already asked all the co-organizers to send me a profile picture to update the PR.

Comment (KTan1226) [reply content/events/2020-paris/_index.md]: Hi @jcsirot- Just checking in on this PR. Is it ready for review?

Comment (jcsirot) [reply content/events/2020-paris/_index.md]: Hi @KTan1226 The website is ok but we are just waiting for the a final confirmation of one the initial sponsors, which takes a little more time than expected due to the current lockdown in France. As soon as we receive the confirmation I will send you a message for the review.

Comment (KTan1226) [reply content/events/2020-paris/_index.md]: Thanks for the update @jcsirot!

Comment (jcsirot) [reply content/events/2020-paris/_index.md:36]: Hello @KTan1226 The PR is ready for review!

Comment (KTan1226) [reply content/events/2020-paris/_index.md:36]: Hi @jcsriot- looks good! Can you have another organizer give a LGTM? I can merge after that.

Comment (jcsirot) [reply content/events/2020-paris/_index.md:36]: @rbenzair please could you give a LGTM before merging

**Comment by Smana** [comment]
Fine for me!

**Comment by rbenzair** [comment]
hi @KTan1226 it is fine for me.

File: content/events/2020-paris/_index.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+---
+title: Paris
+date: '2020-11-04'
```
Comment (KTan1226) [inline content/events/2020-paris/_index.md:3]: Please change to Dec 1, 2020 as a temporary new date. Thanks!

File: content/events/2020-paris/_index.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  image: team/jgarrouste.jpg
+  twitter: https://twitter.com/JeremyGarrouste
+- name: Giovanni Clément
+  image: team/gclement.jpg
+  twitter: https://twitter.com/gioindahouz
+
+organizers_email: organizers-paris@kubernetescommunitydays.org
+local_law_enforcement: "17 or 112"
+medical_emergency: "18 or 112"
+cfp_deadline: '2020-06-28'
```
Comment (KTan1226) [inline content/events/2020-paris/_index.md:65]: Please remove CFP date for now as event is postponed.

**Comment by KTan1226** [comment]
Hi @rbenzair and team- please update the date and CFP date as the event is postponed. Let me know if you have any questions. Thanks!

**Pull request merged**